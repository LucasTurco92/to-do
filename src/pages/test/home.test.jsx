import { render } from '@testing-library/react';
import Home from '../home';

it('When it invokes the home page, should render it properly', ()=>{

    const { container } = render(<Home />);
    
    expect(container).toMatchSnapshot();
});