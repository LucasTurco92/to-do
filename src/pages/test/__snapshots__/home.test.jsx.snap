// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When it invokes the home page, should render it properly 1`] = `
<div>
  <div>
    <div
      class="sc-bdvvtL bwusyA"
    >
      <img
        src="logo192.png"
      />
      <span
        class="sc-gsDKAQ ioyFhy"
      >
        To-Do
      </span>
    </div>
    <div
      class="sc-eCImPb fGXFeQ"
    >
      <div
        class="sc-dkPtRN iLgXIo"
      >
        <div
          class="sc-hKwDye idgETy"
          color="var(--orange)"
        >
          Pending
        </div>
      </div>
      <div
        class="sc-dkPtRN iLgXIo"
      >
        <div
          class="sc-hKwDye fnBAbJ"
          color="var(--green)"
        >
          Completed
        </div>
      </div>
    </div>
  </div>
</div>
`;
